tests/shared/src/test/scala/cats/effect/std/RandomSpec.scala

```diff
@@ -17,38 +17,228 @@
 package cats.effect
 package std
 
+import cats.implicits._
```

Comment:
Nit: `cats.implicits` import is soft-deprecated, the syntax import can fully replace it :)
```suggestion
import cats.syntax.all._
```

Reply:
Noted :)